person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :name
+  attr_reader :id, :age
+
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Good job setting default value for optional variable 👍 , As for the `age` variable , it should not be defined with a default value as it is not a optional variable  , Kindly fix that 👍